packages/e2e-tests/specs/editor/blocks/quote.test.js

```diff
@@ -237,6 +237,7 @@ describe( 'Quote', () => {
 		await page.keyboard.press( 'ArrowLeft' );
 		await page.keyboard.press( 'ArrowDown' );
 		await page.keyboard.press( 'ArrowDown' );
+		await page.keyboard.press( 'ArrowDown' );
```

Comment:
Extra arrow down needed because first the block wrapper will be focused, then the content. The block wrapper is an extra navigation stop as intended.